Lib/ntpath.py

```diff
@@ -647,6 +645,15 @@ def commonpath(paths):
     # Windows XP and non-Windows OS'es will mock _getfinalpathname.
     if sys.getwindowsversion()[:2] >= (6, 0):
         from nt import _getfinalpathname
+
+        def realpath(p):
+            try:
+                p = _getfinalpathname(p)
+                # GetFinalPathNameByHandle returns path in \\?\ syntax
+                # remove leading \\?\ prefix
```

Comment:
I would prefer to be extra careful to ensure that p starts with "\\?\" before removing it.